gatsby-config.js

```diff
@@ -48,6 +48,18 @@ module.exports = {
         prefixDefault: true,
       },
     },
+    {
+      resolve: 'gatsby-plugin-eslint',
+      options: {
+        test: /\.js$|\.jsx$/,
```

Comment:
We don't use `jsx` extension.

```suggestion
        test: /\.js$/,
```

Reply:
Yeah, but I thought it wouldn't hurt. I'll remove this.